src/index.ts

```diff
@@ -410,6 +421,10 @@ class ServerlessAppsyncPlugin {
   async getIntrospection() {
     const apiId = await this.getApiId();
 
+    if (typeof apiId !== 'string') {
+      return;
```

Comment:
just returning is probably not good enough. 
I would show a warning.

In fact I would probably not allow calling any of these commands (assoc domain, etc) from a "child" service stack.

> Please call this command from the service where the AppSync API is created.